<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>256</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>411</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="go_Btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>上一首</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>370</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>下一首</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qiangzhi">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>强制杀死</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>17</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>正在播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cutMusic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>列表选择歌曲</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pidnum">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="currEd">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="filesEd">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>167</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>蓝牙数据数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>files</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bug_le">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>bug</string>
   </property>
  </widget>
  <widget class="QSlider" name="vol_ctl">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>150</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>97</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>音量(切歌时生效)</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
